### ISSUE TITLE
sys.setrecursionlimit breaking other nodes

### ISSUE THREAD

**Issue by demoergo** [WASasquatch/ASTERR#4]
I was hitting a python recursion limit on other nodes and just tracked it down to ASTERR changing the system recursion limit down to 100 by default. No big deal now that I understand what's happening, but I wonder if it would be better for ASTERR the store the system recursion limit and restore it when it's done? I haven't looked at the code to see if that would cause other problems or not. For now I'm just setting it to my system default of 1000 in the config.json.

**Comment by WAS-PlaiLabs** [comment]
Yeah should be able to simply set a var the current recursion limit and then set it back after.
